src/foam/nanos/crunch/ServerCrunchService.java

```diff
@@ -641,28 +646,9 @@ public boolean maybeReopen(X x, String capabilityId) {
 
   public WizardState getWizardState(X x, String capabilityId) {
     var subject = (Subject) x.get("subject");
-
```

Comment:
Note: all of this was removed cause of the following noticed error:

```
2021-10-04T13:54:23.197-0400,qtp864138492-357,ERROR,[Service],crunchService,spid,treviso,user,8006,agent,8007,getJunction,crunch.onboarding.br.parent.international-payments,java.lang.RuntimeException: Capability with id 'crunch.onboarding.br.parent.international-payments' is either unavailabile or does not exist
	at foam.nanos.crunch.ServerCrunchService.buildAssociatedUCJ(ServerCrunchService.java:498)
	at foam.nanos.crunch.ServerCrunchService.getJunctionForSubject(ServerCrunchService.java:405)
	at foam.nanos.crunch.ServerCrunchService.getJunction(ServerCrunchService.java:308)
	at foam.nanos.crunch.ServerCrunchService.getGrantedFor_(ServerCrunchService.java:714)
	at foam.nanos.crunch.ServerCrunchService.getWizardStateFor_(ServerCrunchService.java:686)
	at foam.nanos.crunch.ServerCrunchService.getWizardState(ServerCrunchService.java:660)
```

Reply:
also caused this:

```
2021-10-04T11:49:06.099-0400,qtp864138492-89,ERROR,[Service],crunchService,spid,treviso,user,8006,agent,8007,getJunction,crunch.onboarding.signing-officer-information,java.lang.RuntimeException: Capability with id 'crunch.onboarding.signing-officer-question' is either unavailabile or does not exist
	at foam.nanos.crunch.ServerCrunchService.buildAssociatedUCJ(ServerCrunchService.java:498)
	at foam.nanos.crunch.ServerCrunchService.getJunctionForSubject(ServerCrunchService.java:416)
	at foam.nanos.crunch.ServerCrunchService.getJunctionFor(ServerCrunchService.java:303)
	at net.nanopay.crunch.predicate.SigningOfficerPrivilegesRequested.f(SigningOfficerPrivilegesRequested.java:52)
	at foam.nanos.crunch.CapabilityAvailabilityDAO.f(CapabilityAvailabilityDAO.java:126)
	at foam.nanos.crunch.CapabilityAvailabilityDAO.find_(CapabilityAvailabilityDAO.java:98)
	at foam.dao.ProxyDAO.find_(ProxyDAO.java:351)
	at foam.dao.ProxyDAO.find_(ProxyDAO.java:351)
	at foam.dao.ProxyDAO.find_(ProxyDAO.java:351)
	at foam.dao.ProxyDAO.find_(ProxyDAO.java:351)
	at foam.nanos.auth.AuthorizationDAO.find_(AuthorizationDAO.java:236)
	at foam.dao.ProxyDAO.find_(ProxyDAO.java:351)
	at foam.dao.ProxyDAO.find_(ProxyDAO.java:351)
	at foam.dao.ProxyDAO.find_(ProxyDAO.java:351)
	at foam.dao.ProxyDAO.find_(ProxyDAO.java:351)
	at foam.dao.AbstractDAO.find(AbstractDAO.java:628)
	at foam.nanos.crunch.ServerCrunchService.buildAssociatedUCJ(ServerCrunchService.java:496)
	at foam.nanos.crunch.ServerCrunchService.getJunctionForSubject(ServerCrunchService.java:405)
	at foam.nanos.crunch.ServerCrunchService.getJunction(ServerCrunchService.java:308)
	at foam.nanos.crunch.ServerCrunchService.getGrantedFor_(ServerCrunchService.java:714)
	at foam.nanos.crunch.ServerCrunchService.getWizardStateFor_(ServerCrunchService.java:686)
	at foam.nanos.crunch.ServerCrunchService.getWizardState(ServerCrunchService.java:664)
	```

Reply:
note: line numbers aren't accurate because of random lines of code in file - while testing.
Also note: its still curious that a subject { user: business.id, realUser: business.id } causes the error with subject { user: signingOfficerUser.id, realUser: business.id } ... that is another error... but so far can't find when it does that. With the removal of this code -which doesn't make sense to begin with - the subject is never confused.

Reply:
> a subject { user: business.id, realUser: business.id } causes the error with subject { user: signingOfficerUser.id, realUser: business.id

think removing makes sense to me, but having trouble understanding this part of the explanation, can you please elaborate

Reply:
look at the errors -
2021-10-04T11:49:06.099-0400,qtp864138492-89,ERROR,[Service],crunchService,spid,treviso,user,8006,agent,8007,getJunction

notice:
user 8006, agent,8007

Isn't the user suppose to be 8007 and the agent 8006 ... thats where things look backwards to me.
Although just looked again ... its from the 'LoggerUserInfo' and it is fine.
It defines user to be realUser and agent to be user...